### ISSUE TITLE
standard deviation vs. standard error of the mean

### ISSUE THREAD

**Issue by thawn** [clEsperanto/pyclesperanto_prototype#265]
Some feature extraction functions return the standard deviation (STD). In some cases (i.e. when you want to know the accuracy of your measurement rather than the variance of your data) it may be useful to return the standard error of the mean (SEM) instead (or in addition)

SEM = STD/ sqrt(n) where n = number of measurements.

**Comment by haesleinhuepf** [comment]
Yeah, that's a good point. From my perspective, there are some functions where it might maybe make sense to implement SEM versions from. Also check [documentation](https://clesperanto.github.io/pyclesperanto_prototype/docs/_build/html/). 

Tick all that apply for you:

Filters:
* [ ] standard_deviation_box
* [ ] standard_deviation_sphere

Graph based functions;
* [ ] generate_standard_deviation_intensity_between_points_matrix
* [ ] standard_deviation_intensity_map
* [ ] standard_deviation_of_distal_neighbors_map
* [ ] standard_deviation_of_n_most_touching_neighbors_map
* [ ] standard_deviation_of_n_nearest_neighbors_map
* [ ] standard_deviation_of_proximal_neighbors_map
* [ ] standard_deviation_of_touch_portion_within_range_neighbors_map
* [ ] standard_deviation_of_touching_neighbors
* [ ] standard_deviation_of_touching_neighbors_map
* [ ] standard_deviation_touch_portion
* [ ] standard_deviation_touch_portion_map

Projections:
* [ ] standard_deviation_z_projection

We could then implement those similar to how variance and standard deviation are connected, e.g. here:;
https://github.com/clEsperanto/pyclesperanto_prototype/blob/006379e7302cc0f76a571b5cdf3fb3962c9461c9/pyclesperanto_prototype/_tier2/_standard_deviation_box.py#L33-L35

I would keep it minimal if possible for now, and/or postpone implementing anything until we find a project where we need it. Also, e.g. standard-error-filter is not common afaik. But maybe it makes a lot of sense!

**Comment by thawn** [comment]
I cannot check the boxes, but I think SEM could be useful 
* for all of the measurements. 
* For the Graph based functions, I am not so sure. I would need to understand them better.
* For the projections and filters, the SEM is basically a constant factor away from the STD and probably not useful.

**Comment by haesleinhuepf** [comment]
> I cannot check the boxes
You should have received an invitation @thawn  👋
